bin/apisix

```diff
@@ -854,12 +854,21 @@ local openresty_args = [[openresty  -p ]] .. apisix_home .. [[ -c ]]
                        .. apisix_home .. [[/conf/nginx.conf]]
 
 function _M.start(...)
+    -- check running
+    local pid_path = apisix_home .. "/logs/nginx.pid"
+    local pid, err = read_file(pid_path)
+    if pid then
+        print("APISIX is running...")
```

Comment:
It is not enough to judge whether the file exists.

If the nginx process is killed by `kill -9`, there will still be a local `pid` file.

We need to determine whether the process exists.

Reply:
and we can add a test case to check this case.